conf/environments.js

```diff
@@ -102,6 +102,11 @@ module.exports = {
             ecmaVersion: 6
         }
     },
+    es7: {
```

Comment:
Gotcha, that's the piece I was missing. If "es6" is an outlier than it definitely doesn't necessarily make sense to continue the pattern with "es7".